docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -172,50 +154,6 @@ async function postFeedbackOnSlack(data) {
   */
 }
 
-async function getUserFeedback(id) {
-  const env = location.hostname === 'mui.com' ? 'prod' : 'dev';
-  const URL = `${process.env.FEEDBACK_URL}/${env}/feedback/${id}`;
-
-  try {
-    const response = await fetch(URL, {
-      method: 'GET',
-      cache: 'no-store',
-      referrerPolicy: 'origin',
-    });
-    return response.json();
-  } catch (error) {
-    console.error(error);
-    return null;
-  }
-}
-
-async function submitFeedback(page, rating, comment, language, commentedSection, productId) {
-  const resultSlack = await postFeedbackOnSlack({ rating, comment, commentedSection, productId });
-
-  if (rating !== undefined) {
-    const resultVote = await postFeedback({
-      id: getCookie('feedbackId'),
-      page,
-      rating,
-      comment,
-      version: process.env.LIB_VERSION,
-      language,
-    });
-    if (resultVote) {
-      document.cookie = `feedbackId=${resultVote.id};path=/;max-age=31536000`;
-      setTimeout(async () => {
-        const userFeedback = await getUserFeedback(resultVote.id);
-        if (userFeedback) {
-          document.cookie = `feedback=${JSON.stringify(userFeedback)};path=/;max-age=31536000`;
-        }
```

Comment:
This is some logic used to keep in memory the user already commented a page.

I assume without this line the `getCookie('feedback')` in the `getCurrentRating()` is useless and most of the fucntion using `getCurrentRating` are useless too

Reply:
It's currently also broken as the `postFeedback` function throws. I'll move this up